README.md

```diff
@@ -16,28 +16,146 @@ This simple implementation stores bubbles as directories on the server's local f
 
-## Usage
+This guide explains the settings available in `config.json` for configuring the Bubble server with multiple blockchain networks.
+
+## Configuration Options
+
+### Server Port (`serverPort`)
+The main server port that your Bubble server will listen on, such as `3000`.
+
+### Chains (`chains`)
+Define each blockchain network with the following options:
+
+- **name**: A friendly name for each blockchain network (e.g., "Ethereum Mainnet", "Goerli Testnet", "Ganache Local").
+- **chainId**: The unique chain ID for the blockchain network (e.g., `1` for Ethereum mainnet, `5` for Goerli testnet, `1337` for a local Ganache network).
+- **web3Url**: The URL for the Web3 provider:
+  - For Infura or another Web3 API provider, use the respective URL with an API key (e.g., `https://mainnet.infura.io/v3/YOUR_INFURA_PROJECT_ID`).
+  - For a local Ethereum node, use its RPC endpoint (e.g., `http://localhost:8545`).
+  - For a local Ganache instance, use `http://127.0.0.1:8545`.
+- **endpoint**: The endpoint path for this blockchain, used for routing requests (e.g., `/eth-mainnet`).
+- **storageDirectory**: The path for local storage of bubbles on this blockchain network.
+
+### Root Path (`rootPath`)
+The base directory path where bubble storage directories will be created if not specified explicitly under each chain's `storageDirectory`.
+
+---
+
+### Example `config.json`
+
+```json
+{
+  "version": "0.2.0",
+  "port": 3000,
+  "hostname": "localhost",
+  "https": {
+    "active": false,
+    "key": "<PATH_TO_PRIVATE_KEY>",
+    "cer": "<PATH_TO_CERTIFICATE>"
+  },
+  "debugOn": false,
+  "traceOn": true,
+  "v2": {
+    "chains": [
+      {
+        "endpoint": "base-goerli",
+        "chainId": 84531,
+        "rootPath": "./bubbles/base-goerli",
+        "web3Url": "https://goerli.infura.io/v3/YOUR_INFURA_PROJECT_ID",
+        "throttling": {
+          "maxRequests": 25,
+          "window": 1000
+        }
+      },
+      {
+        "endpoint": "ethereum",
+        "chainId": 1,
+        "rootPath": "./bubbles/ethereum",
+        "web3Url": "https://mainnet.infura.io/v3/YOUR_INFURA_PROJECT_ID"
+      },
+      {
+        "endpoint": "polygon",
+        "chainId": 137,
+        "rootPath": "./bubbles/polygon",
+        "web3Url": "https://polygon-amoy.infura.io/v3/YOUR_INFURA_PROJECT_ID"
+      },
+      {
+        "endpoint": "avalanche",
+        "chainId": 43114,
+        "rootPath": "./bubbles/avalanche",
+        "web3Url": "https://avalanche-fuji.infura.io/v3/YOUR_INFURA_PROJECT_ID"
+      },
+      {
+        "endpoint": "sepolia",
+        "chainId": 11155111,
+        "rootPath": "./bubbles/sepolia",
+        "web3Url": "https://sepolia.infura.io/v3/YOUR_INFURA_PROJECT_ID"
+      },
+      {
+        "name": "Local Ganache",
+      "chainId": 1337,
+      "web3Url": "http://127.0.0.1:8545",
+      "endpoint": "/ganache",
+      "storageDirectory": "./storage/ganache"
```

Comment:
`storageDirectory` isn't an option. Should be `rootPath`.

Also, fix indentation.